### PR TITLE
[TechDocs] Integrate add-ons with create-app

### PR DESCRIPTION
**Pull request by camilaibs**
## Hey, I just made a Pull Request!

This pull request integrates TechDocs add-ons with the app package so add-ons are configured when creating an app using the Backstage CLI:

- [x] Configure all add-on dependencies
- [x] Register ReportIssue in the App template
- [x] Create an end-to-end test by testing the add-on
image
image
- [x] Update getting started documentation
image
- [x] Update report issue documentation
image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: packages/create-app/templates/default-app/packages/app/src/App.tsx
Comment (Rugvip) [inline packages/create-app/templates/default-app/packages/app/src/App.tsx:22]: Gotta say I'm not super fond of adding alpha features into the `create-app` template. Would a middle-ground perhaps be to add a comment with a link to docs for how to set it up or something like that instead?

Thoughts on this @backstage/maintainers / @backstage/techdocs-core?

Comment (iamEAP) [reply packages/create-app/templates/default-app/packages/app/src/App.tsx:22]: FWIW, I'm (we're?) hoping to remove the `alpha` from this before the 1.2 release.

File: docs/features/techdocs/getting-started.md
Comment (iamEAP) [inline docs/features/techdocs/getting-started.md:92]: Maybe now that you've made the documentation page for the `<ReportIssue />` addon itself _really_ nice, we can get remove it from here / remove this section to keep the "getting started" flow a little smoother?

**Review by iamEAP — COMMENTED** [review]
Some documentation style comments for you @camilaibs, but looking really good!

**Comment by Rugvip** [comment]
#11150 should fix the CI issue

**Review by iamEAP — APPROVED** [review]
Niiiice!

File: packages/create-app/templates/default-app/packages/app/src/App.tsx
Comment (camilaibs) [reply packages/create-app/templates/default-app/packages/app/src/App.tsx:22]: @Rugvip, just to let you know that we will wait for the next release to avoid adding an alpha feature 🙂 , thanks for your review!

Comment (camilaibs) [reply packages/create-app/templates/default-app/packages/app/src/App.tsx]: @iamEAP and @Rugvip, version alpha removed!

**Review by iamEAP — APPROVED** [review]
Good to go!

One more look @Rugvip?  We got rid of the `/alpha` imports.

**Review by Rugvip — APPROVED** [review]
Nice! 👍 

:shipit:

**Pull request merged**
